apps/wing-console/console/ui/src/features/logs-pane/console-logs.tsx

```diff
@@ -234,6 +262,16 @@ export const ConsoleLogs = memo(
   }: ConsoleLogsProps) => {
     const { theme } = useTheme();
 
+    const useLongDateFormat = useMemo(() => {
+      if (!logs[0]?.timestamp) {
+        return false;
+      }
+      const now = new Date();
+      const logDate = new Date(logs[0].timestamp);
+
+      return logDate.getDate() !== now.getDate();
+    }, [logs]);
+
```

Comment:
I think we could improve this implementation. The problem I see now is that this hook will be executed every time there's a new log entry, but as soon as we know we need to use the long date format, we don't have to check again.

I propose using a `const [useLongDateFormat, setUseLongDateFormat] = useState(false);` and an effect that depends on `useLongDateFormat, logs`. If `useLongDateFormat` is true, no need to check; otherwise, check if the date in the timestamps is different and update with `setUseLongDateFormat` as appropiate.